tests/cli.rs

```diff
@@ -509,6 +509,40 @@ fn request_binary_detection() {
         "#});
 }
 
+#[test]
+fn timeout() {
+    let mut cmd = get_command();
+    cmd.arg("--timeout=0.01")
+        .arg("GET")
+        .arg("https://httpbin.org/delay/0.5");
```

Comment:
Could you use a `MockServer` with the [`delay`](https://docs.rs/httpmock/0.5.8/httpmock/struct.Then.html#method.delay) method instead? Then we don't need httpbin for these tests.

And maybe 0.05 instead of 0.5 would be enough, that would make the tests a bit faster.

Reply:
I tried it, but it seems difficult for me. I'm sorry.

Reply:
I can't make it work either, I guess httpmock doesn't delay in the right way.